docs/emails.rst

```diff
@@ -90,9 +90,25 @@ Models
 
     String tag that you can define during sending SMS message.
 
-  .. attribute:: failed
+  .. attribute:: number_of_send_attempts
 
-    Returns ``True`` if SMS ended in ``ERROR`` state.
+    Number of sent attempts. Value is set only when batch sending is used.
+
+  .. attribute:: external_id
```

Comment:
Did you know that you can write docstrings and then autogenerate reference documentation from it? IMHO it is better since:
- the doc is directly at class it refers to (easy maintenance when something changes - everything at one place)
- you don't need to search it when you work with the class
- from generated documentation, you can easily click to see the method/attribute in the source code

Check this out, this section is completely autogenerated, it works very well:
https://django-pynotify.readthedocs.io/en/latest/reference.html